pyEpiabm/pyEpiabm/property/household_foi.py

```diff
@@ -86,8 +87,19 @@ def household_foi(infector, infectee, time: float):
             carehome_scale_susc = Parameters.instance()\
                 .carehome_params["carehome_resident_household_scaling"]
         seasonality = 1.0  # Not yet implemented
+        false_pos = 1 / (1 - Parameters.instance().
+                         false_positive_rate)
+        vacc_inf_drop = 1
+        if infector.is_vaccinated:
+            vacc_params = Parameters.instance()\
+                .intervention_params['vaccine_params']
+            if time > (infector.date_vaccinated +
+                       vacc_params['time_to_efficacy']):
+                vacc_inf_drop *= (1 - vacc_params['vacc_inf_drop'])
```

Comment:
I followed the seasonality, false positive, and carehome scaling and scaled infectiousness within the foi function for vaccination.

Reply:
Sorry I'm not quite sure what you mean here (as vaccination doesn't have its own FOI). I agree seasonality and false positive should be in the FOI directly, but I think if we scale infectiousness we should do that in the infectiousness function (`household_inf()`)

Reply:
I'm happy to change this but should do the same for other scaling factors as well. Vaccination scales infectiousness only but within the foi function i.e. the scaling is done on the infectiousness value, and then the foi is still only calculated as infectiousness * susceptibility. This is also how seasonality, isolation, and quarantine are done (so seasonality still scales infectiousness). I agree it may be neater to move these out of the foi function and into the inf or susc though.